src/Rings/mpoly-localizations.jl

```diff
@@ -1102,7 +1102,7 @@ mutable struct LocalizedBiPolyArray{BRT, BRET, RT, RET, MST}
   # The polynomial ring on the singular side
   singular_ring::Singular.PolyRing
   # The ordering used for the above singular ring.
-  ordering::Symbol
+  ordering::Orderings.AbsOrdering
```

Comment:
This is not a concrete type. Do we want to introduce a type parameter `OrderingType` for this?

Reply:
I followed the implementation in `BiPolyArray` here. If you want me to, I can add a type parameter.

Reply:
Let's see about this. One reason against doing that is, that then also the value type of the dictionary for caching groebner bases can hardly be fixed a priori. So it might even be better to keep it this way.